L/LibTracyClient/LibTracyClient@0.9.0/bundled/patches/libTracyClient-no-crash-handler.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/public/client/TracyProfiler.cpp b/public/client/TracyProfiler.cpp
```

Comment:
Should this patch be upstreamed?

Reply:
It's already available on master upstream, but unfortunately didn't make it into the v0.9 release

Reply:
Ok, perfect, since this isn't a git patch I couldn't guess where it was coming from 🙂